internal/http/services/owncloud/ocs/conversions/main.go

```diff
@@ -361,6 +361,7 @@ func publicSharePermissions2OCSPermissions(sp *link.PublicSharePermissions) Perm
 }
 
 // TODO sort out mapping, this is just a first guess
+// public link permissions to OCS permissions
 func Permissions2OCSPermissions(p *provider.ResourcePermissions) Permissions {
```

Comment:
On an organization basis we should get rid of this conversions package in favor of letting the package do the conversions. What I mean is moving `Permissions2OCSPermissions` to the `ocs` package itself. And so on with the respective conversions.

Reply:
sounds good, please post your ideas in https://github.com/owncloud/ocis-reva/issues/292 or create another tech debt ticket